build-repo.sh

```diff
@@ -135,11 +135,8 @@ ext_deps=$(find ${SRCDIR} -name "*.rosinstall" || true)
 tmp_ws=$(mktemp -d)
 touch ${tmp_ws}/.rosinstall
 for dep in ${ext_deps}; do
-  wstool merge -y -t ${tmp_ws} $dep
+  vcs import ${tmp_ws}/.rosinstall ${ext_pkg_option} --workers 4
 done
```

Comment:
Seems that `ext_deps` are ignored and packages in base `.rosinstall` is tried to be installed multiple times.

Reply:
:bow: